student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom)
+    super(age)
+    @classroom = classroom
```

Comment:
- [ ] you will also need to initialize the name and parent permission in this class.